05/styles/desktop.css

```diff
@@ -0,0 +1,95 @@
+body {
+    display:flex;
+    flex-direction: column;
+    align-items: center;
+}
+
+.page__container--center {
+    width: 1000px;
```

Comment:
Lepiej użyć `max-width`, wtedy jeśli strona ma `900px` to nie ma paska przewijania u dołu tylko strona "się zmniejsza".
To rozwiązanie jest bardziej zgodne z RWD :)